### PR TITLE
Fix documentation

### PR DESCRIPTION
**Pull request by fdrouet**
Fix and improve the setup documentation

- fix some ports in the doc
- fix the way to start Jenkins agent with JWS
- add images for simpler understanding

### CODE REVIEW

File: doc/getting-started.adoc [hunk 1]
```diff
@@ -83,14 +83,34 @@ $ git branch -vv
 
 === Configure Jenkins
 
-You should have a Jenkins agent already created.
+Jenkins after the first startup of Jenkins container you will have to finish the install in your browser.
```
Comment (bourasom) [inline doc/getting-started.adoc]:
```suggestion
After the first startup of Jenkins container you will have to finish the install in your browser.
```

File: doc/getting-started.adoc [hunk 2]
```diff
@@ -83,14 +83,34 @@ $ git branch -vv
...
+
+* Go to http://localhost:8888/ 
+* fill the (generated) admin password which is available in jenkins startup log
+* choose to select the Jenkins Plugins you want to install
+* unselect ALL plugins from the plugin install selection screen
+
+You should have a Jenkins agent already created but you need need to start it
```
Comment (bourasom) [inline doc/getting-started.adoc]:
```suggestion
You should have a Jenkins agent already created but you need to start it
```

Comment (fdrouet) [reply doc/getting-started.adoc]: Created = configured / declared in Jenkins
Started = ... you need to start it on your host machine with JavaWebStart 😄

File: doc/getting-started.adoc (hunk 1 above)
Comment (fdrouet) [reply doc/getting-started.adoc]: you should have putted your suggestion at the right place to let me accept it ;-)
```suggestion
After the first startup of Jenkins container you will have to finish the install in your browser.
```

**Review by bourasom — APPROVED** [review]
LGTM

**Pull request merged**